### PR TITLE
ci(travis): pip, platform upgrades

### PR DESCRIPTION
**Pull request by joshuarli**
Pins pip to a more recent version, and upgrades CI to use bionic because wanted to see what it would take to bump CI to bionic.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
-install-yarn:
-	@echo "--> Installing Node dependencies"
-	@hash yarn 2> /dev/null || npm install -g yarn
-	# Use NODE_ENV=development so that yarn installs both dependencies + devDependencies
-	NODE_ENV=development yarn install --ignore-optional
-
-install-tests: develop
-	pip install .[tests]
+ensure-pip-version:
+	pip install "pip>=19.2.1,<20"
```
Comment (EvanPurkhiser) [inline Makefile]: Can we lock to a specific pip version

Comment (joshuarli) [reply Makefile]: I feel like a major pin > 19.2 is good enough, but sure.

Comment (EvanPurkhiser) [reply Makefile]: It's probably fine, but we've been pinning to explicit versions everywhere else.

Comment (joshuarli) [reply Makefile]: getsentry is >=19,<20, and sentry doesn't have pip pinned (just at a cursory glance), so actually i'm just gonna leave this as is.

**Comment by joshuarli** [comment]
Ok, gonna put this on the backburner for now. For some reason travis isn't installing apt packages. I've got a feeling it has something to do with:

```
sudo: unable to resolve host travis-job-489e7ef7-c8cf-41f9-bcd9-2f1cae7ce05c
sudo: unable to resolve host travis-job-489e7ef7-c8cf-41f9-bcd9-2f1cae7ce05c
sudo: unable to resolve host travis-job-489e7ef7-c8cf-41f9-bcd9-2f1cae7ce05c
```

The "Installing APT Packages" is being completely skipped.

**Comment by joshuarli** [comment]
Pre-emptively closing this since sentry-plugins will be moved into sentry soon.

**Pull request closed without merging**